### ISSUE TITLE
Add a mechanism for cancelling the login operation

### ISSUE THREAD

**Issue by tst32** [microsoft/vscode-azure-account#468]
it is not an important bug, just want an option to cancel gracefully login request
Does this occur consistently? Yes
Repro steps:

1. When do ocassionaly pressed azure func shortcut in vscode
2. i don't wnat to login but want to close azure func i have no option bu wait for timeout error

Action: azure-account.login
Error type: Error
Error Message: Timeout waiting for code

Version: 0.10.0
OS: linux
OS Release: 5.4.0-100-generic
Product: Visual Studio Code
Product Version: 1.64.2
Language: en

<details>
<summary>Call Stack</summary>

```
Timeout.<anonymous> extension.bundle.js:2:2321688
listOnTimeout timers.js:554:17
processTimers timers.js:497:7
```

</details>

**Comment by wwlorey** [comment]
Hi @tst32, does running the `Azure: Sign Out` command via the [command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) accomplish this for you?

**Comment by tst32** [comment]
![image](https://user-images.githubusercontent.com/23115742/155007241-49eece03-cd66-429b-8139-ce6894f77fa8.png)
No, <kbd> Cntrl </kbd> <kbd> Shift </kbd> <kbd> P </kbd> `Azure sign out` does nothing

**Comment by wwlorey** [comment]
Let's add a cancellable, long-running notification that's shown for the duration of the sign in process.

**Comment by wwlorey** [comment]
@tst32 FYI we released this feature in [v0.10.1](https://github.com/microsoft/vscode-azure-account/releases/tag/v0.10.1).

**Comment by tst32** [comment]
well implemented!
